build.sh

```diff
@@ -165,4 +168,16 @@ if [ "${2}" == "--push" ] || [ "${2}" == "--push-only" ] ; then
   echo "⏫ Pushing '${DOCKER_TAG}"
   $DOCKER_CMD push "${DOCKER_TAG}"
   echo "✅ Finished pushing the Docker image '${DOCKER_TAG}'."
+  if [ "${TAG:0:1}" == "v" ]; then
```

Comment:
I suggest to use bash's built-in regex here:

```bash
# TAG=v2.5.5

if [[ "${TAG}" =~ v([0-9]+)\.([0-9]+)\.[0-9]+ ]]; then
  MAJOR=${BASH_REMATCH[1]}
  MINOR=${BASH_REMATCH[2]}
  DOCKER_SHORT_TAG="${DOCKER_SHORT_TAG-${DOCKER_ORG}/${DOCKER_REPO}:${MAJOR}.${MINOR}}"
  # ...
```

This reduced the need for `cut` to be available and behaving in a certain manner.

Reply:
That is a good idea. I will make the change.